model/rpc/spans.yaml

```
@@ -35,6 +35,48 @@ groups:
       - ref: rpc.system
         requirement_level: required
 
+  - id: span.rpc.connect_rpc.client
+    type: span
+    stability: development
+    brief: This span represents an outgoing Remote Procedure Call (RPC).
+    note: |
+      `rpc.system` MUST be set to `"connect_rpc"` and SHOULD be provided **at span creation time.**
+
+      **Span name:** refer to the [Span Name](/docs/rpc/rpc-spans.md#span-name) section.
+
+      **Span kind** MUST be `CLIENT`.
+    extends: rpc_service
+    span_kind: client
+    attributes:
+      - ref: rpc.connect_rpc.error_code
+        requirement_level:
+          conditionally_required: if available.
+      - ref: rpc.connect_rpc.request.metadata
+        requirement_level: opt_in
+      - ref: rpc.connect_rpc.response.metadata
+        requirement_level: opt_in
+
+  - id: span.rpc.connect_rpc.server
+    type: span
+    stability: development
+    extends: rpc_service.server
+    span_kind: server
+    brief: This span represents an incoming Remote Procedure Call (RPC).
+    note: |
+      `rpc.system` MUST be set to `"connect_rpc"` and SHOULD be provided **at span creation time.**
+
+      **Span name:** refer to the [Span Name](/docs/rpc/rpc-spans.md#span-name) section.
+
+      **Span kind** MUST be `SERVER`.
+    attributes:
+      - ref: rpc.connect_rpc.error_code
+        requirement_level:
+          conditionally_required: if available.
```

any need for this change?

```suggestion
          conditionally_required: If response is not successful and if error code available.
```